www/src/record/record.list.html

```diff
@@ -1,4 +1,4 @@
-<ion-view view-title="Registros">
+<ion-view view-title="{{ vm.research.name }}">
```

Comment:
Any of the two solutions it's better hahaha Choose one ... I personally like $ionicNavBarDelegate because it's more native like.